app/invocation/invocation_action_card.tsx

```diff
@@ -639,6 +644,31 @@ export default class InvocationActionCardComponent extends React.Component<Props
       });
   }
 
+  private onClickCopySnapshotKey(vmMetadata: firecracker.VMMetadata) {
+    const snapshotKey = this.getSnapshotKeyForSnapshotID(vmMetadata);
+    copyToClipboard(JSON.stringify(snapshotKey));
+    alert_service.success("Snapshot key copied to clipboard");
+  }
+
+  private onClickCopyRemoteBazelCommand(vmMetadata: firecracker.VMMetadata) {
+    const snapshotKey = this.getSnapshotKeyForSnapshotID(vmMetadata);
+    const snapshotKeyJSON = JSON.stringify(snapshotKey);
+    const cmd = `bb remote --key='${snapshotKeyJSON}' --script=""`
```

Comment:
nit: update `--key` arg for rename, also maybe give an example `--script` to make it clear that it accepts bash code?

```suggestion
    const cmd = `bb remote --run_from_snapshot='${snapshotKeyJSON}' --script="bazel version"`
```

Reply:
Good catch. Fixed